test/ably/realtimeconnection_test.py

```diff
@@ -330,5 +330,27 @@ def once_disconnected(state_change):
         assert state_change.current == ConnectionState.DISCONNECTED
         assert state_change.reason.code == 80003
         assert state_change.reason.status_code == 408
+        await ably.close()
+
+    async def test_fallback_host(self):
+        fallback_host = 'sandbox-realtime.ably.io'
+        ably = await RestSetup.get_ably_realtime(realtime_host="iamnotahost", disconnected_retry_timeout=1000,
```

Comment:
fallback retry should happen immediately, so you shouldn't need to set disconnected_retry_timeout to 1s here